source/opt/debug_info_manager.h

```diff
@@ -148,6 +148,10 @@ class DebugInfoManager {
   // Erases |instr| from data structures of this class.
   void ClearDebugInfo(Instruction* instr);
 
+  // Returns true if |instr| is a DebugDeclare or a DebugValue with Deref
+  // that is used for a declaration.
+  bool IsUsedForDebugDeclare(Instruction* instr);
```

Comment:
I don't follow.  Does this mean that the instruction acts as a declaration point?  Used for `DebugDeclare` means to me a `DebugDeclare`.  Perhaps `IsDebugInfoDeclaration`?  See my comment below.

Reply:
I just changed it to `IsDebugDeclare` that returns true if `instr` is a debug declaration instruction.